src/GuppyDataExplorer/ExplorerTopMessageBanner/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@gen3/ui-component/dist/components/Button';
+import './ExplorerTopMessageBanner.css';
+import { checkForNoAccessibleProject } from '../GuppyDataExplorerHelper';
+import { GuppyConfigType } from '../configTypeDef';
+
+class ExplorerTopMessageBanner extends React.Component {
+  render() {
+    return (
+      <div className={this.props.className}>
+        {
+          (this.props.tierAccessLevel === 'regular' && checkForNoAccessibleProject(this.props.accessibleFieldObject, this.props.guppyConfig.accessibleValidationField)) ? (
+            <div className='top-message-banner'>
+              <div className='top-message-banner__button-wrapper'>
+                <Button
+                  label='Get Access'
+                  className='top-message-banner__button'
+                  buttonType='default'
+                  onClick={
+                    (this.props.getAccessButtonLink) ? (
+                      () => { window.open(this.props.getAccessButtonLink); }
+                    ) : (() => {})
+                  }
+                />
+              </div>
+              <span className='top-message-banner__normal-text'>To protect data security, you can only narrow the cohort down to </span>
```

Comment:
Can we change this message? I think "to protect data security" sounds awkward
Maybe something like "Due to lack of access, you are only able to narrow the cohort down to X ${whatever they call the records}. Please request additional access if necessary"

I think we also shouldn't hardcode "subjects" - some groups say cases, some patients, etc